.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,20 @@
-Please follow the steps below to help us help you.
+Is this a question, feature request, or bug report?
+
+**QUESTION**
+
+Have you checked out our community forum at https://forum.cockroachlabs.com/? Please consider asking your question there, as it might benefit other members of our community.
```

Comment:
I'm good with this as is, but we might want to work refer users to the documentation as well. Perhaps something like:

Have you checked our documentation at https://cockroachlabs.com/docs/? If yes, and you haven't found an answer there, please consider asking your question in our community forum at https://forum.cockroachlabs.com/.